internal/watcher/health/nginx_health_watcher_operator.go

```diff
@@ -39,7 +36,7 @@ func (nhw *NginxHealthWatcher) Health(ctx context.Context, instance *mpi.Instanc
 	}
 
 	proc, err := nhw.processOperator.Process(ctx, instance.GetInstanceRuntime().GetProcessId())
-	if errors.Is(err, process.ErrorProcessNotRunning) {
+	if nginxprocess.IsNotRunningErr(err) {
```

Comment:
I'm matching the existing behavior here, but there's a slim chance that `instance.GetInstanceRuntime().GetProcessId()` is a running process but not an NGINX process. Should we add another branch using `nginxprocess.IsNotNginxErr` to catch that case?